Formula/vdirsyncer.rb

```diff
@@ -1,8 +1,9 @@
+# coding: utf-8
```

Comment:
This still shouldn't be necessary. `utf-8` is set for all formulae files automatically.